src/mumble/Settings.cpp

```diff
@@ -895,7 +895,11 @@ void Settings::legacyLoad(const QString &path) {
 	LOAD(qsTTSLanguage, "tts/language");
 
 	// Network settings
-	LOAD(bTCPCompat, "net/tcponly");
+	LOAD(bUdpDisabled, "net/tcponly");
+	LOAD(bUdpStreamInboundOnly, "net/udpin");
+	LOAD(bUdpStreamOutboundOnly, "net/udpout");
+	LOAD(bUdpStreamBidirectional, "net/udpboth");
+	LOAD(bUdpStreamAuto, "net/udpauto");
```

Comment:
These are new settings, so they should not be loaded as legacy settings. This function is only needed to provide backwards compatibility for migrating the old INI format to the new JSON format. All settings that are added only after the introduction of the JSON format should not be added here to the legacy load function.
```suggestion
	LOAD(bUdpDisabled, "net/tcponly"); 
```